src/ConfigurationTrait.php

```diff
@@ -92,10 +92,19 @@ public function getConfig()
                     'port' => isset($config['port']) ? $config['port'] : null,
                     'name' => $config['database'],
                     'charset' => isset($config['encoding']) ? $config['encoding'] : null,
-                    'unix_socket' => isset($config['unix_socket']) ? $config['unix_socket'] : null,
                 ]
             ]
-        ]);
+        ];
+
+        $mysqlAttributesArray = [ 'ssl_ca', 'ssl_cert', 'ssl_key' ];
+
+        foreach ($mysqlAttributesArray as $value) {
+            if (isset($config[$value])) {
+                $configArray['environments']['default']['mysql_attr'.$value] = $config[$value];
```

Comment:
Concatenate is missing spaces e.g : ``'mysql_attr' . $value``
It is missing an underscore for this to work :

```php
configArray['environments']['default']['mysql_attr_' . $value] = $config[$value];
```

[(Ref in phinx)](https://github.com/robmorgan/phinx/blob/0.4.x-dev/src/Phinx/Db/Adapter/MysqlAdapter.php#L104-L108)